src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -39,15 +44,20 @@ open opposite
 
 namespace algebraic_geometry
 
+/--
+`$$Spec R$$, just as a topological space.
```

Comment:
```suggestion
$Spec R$, just as a topological space.
```
Single `$` signs should work for inline LaTeX, cf. https://leanprover-community.github.io/mathlib_docs/analysis/mean_inequalities.html and https://github.com/leanprover-community/mathlib/blob/master/src/analysis/mean_inequalities.lean